entgql/template/enum.tmpl

```diff
@@ -8,6 +8,10 @@ in the LICENSE file in the root directory of this source tree.
 
 {{ define "meta/additional/graphql-enum" }}
 
+import (
+	"github.com/99designs/gqlgen/graphql"
+)
```

Comment:
```suggestion
import	"github.com/99designs/gqlgen/graphql"
```

Reply:
it isn't working. because we are in another template.